lava_test_plans/testcases/mmtests-fio-randread-async-randwrite.yaml

```diff
@@ -0,0 +1,6 @@
+{% extends "testcases/master/template-mmtests.jinja2" %}
+
+{% set testname = 'fio' %}
```

Comment:
We need to set test_name to something too, probably something like
test_name = 'mmtests-fio-randread-async-randwrite'